src/averagePerceptronTagger.jl

```diff
@@ -0,0 +1,306 @@
+using DataStructures
+using Random
+using BSON
+
+"""
+This file contains the Average Perceptron model and Perceptron Tagger
+which was original implemented by Matthew Honnibal.
+
+The model learns by basic perceptron algorithm
+but after all iterations weights are being averaged
+
+AVERAGE PERCEPTRON MODEL
+
+This struct contains the actual Average Perceptron Model
+"""
+
+mutable struct AveragePerceptron
+    classes :: Set
+    weights :: Dict
+    _totals :: DefaultDict
+    _tstamps :: DefaultDict
+    i :: Int64
+    START :: Array
+
+    function AveragePerceptron()
+        new(Set(), Dict(), DefaultDict(0), DefaultDict(0), 1, ["-START-", "-START2-"])
+    end
+end
+
+"""Predicting the class using current weights by doing Dot-product of features

```

Comment:
Point 8 on what official julia docs say about proper documentation - https://docs.julialang.org/en/v1/manual/documentation/index.html
- Starting and Ending quotes `"""` should be on a separate line by themselves.